pallets/dkg-proposal-handler/src/mock.rs

```diff
@@ -169,6 +175,26 @@ where
 	}
 }
 
+type IdentificationTuple = (AccountId, AccountId);
+type Offence = crate::DKGMisbehaviourOffence<IdentificationTuple>;
+
+parameter_types! {
+	pub static Offences: Vec<(Vec<AccountId>, Offence)> = vec![];
```

Comment:
Would it maybe make more sense to use:

```HashMap<Offence, Vec<AccountId>>```

Reply:
Hashmap is not recommended in substrate no_std env
https://substrate.stackexchange.com/questions/6929/can-i-store-a-hashmap-in-storagemap